src/device/router_device_routing.erl

```diff
@@ -667,7 +667,7 @@ maybe_multi_buy(Offer, Attempts, Device) ->
     end.
 
 -spec lookup_mb(binary()) ->
-    {ok, binary(), non_neg_integer(), non_neg_integer()} | {error, not_found}.
+    {ok, binary(), non_neg_integer(), integer()} | {error, not_found}.
 lookup_mb(PHash) ->
```

Comment:
The text of your commit message might be useful as a code comment, but perhaps there's a better place to put it than here?

```suggestion
%% use -1 to mean deny more
lookup_mb(PHash) ->
```